assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * Idea Hub DashboardCTA styles
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-idea-hub-dashboardcta {
+		background-color: $c-white;
+		display: flex;
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__content {
+		display: inline-block;
+		padding: $grid-gap-desktop;
+
+		h5 {
+			font-family: $f-secondary;
+			font-size: 22px;
+			font-weight: $fw-secondary-normal;
+			margin: 0;
+			text-align: left;
+		}
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__icon {
+		display: inline-block;
+		padding: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__icon svg {
+		background-color: rgba($c-idea-hub-orange, 0.2);
+		background-size: 144px 144px;
+		border-radius: 50%;
+		height: 144px;
+		padding: 32px;
+		width: 144px;
```

Comment:
Why not set the `height` and `width` on the `BulbIcon` instead? It currently has `10` set for both. Dimensions like this are best set on the component.

Reply:
I think my thought process was that since it is sitting inline with the text I wanted to use REM so that it increases / decreases nicely along with the font size. 
I've removed this and added the fixed height to the `BulbIcon` as suggested.